Extension/src/githubAPI.ts

```diff
@@ -100,8 +100,8 @@ function vsixNameForPlatform(info: PlatformInformation): string {
             case 'darwin': return 'cpptools-osx.vsix';
             default: {
                 switch (platformInfo.architecture) {
-                    case 'x86': return 'cpptools-linux32.vsix';
                     case 'x86_64': return 'cpptools-linux.vsix';
+                    default: return 'cpptools-linux32.vsix';
```

Comment:
Should we allow auto-updating when we are not 100% certain of the OS architecture?  I think I would rather fail here - just to be safe.

Reply:
Okay. If that's the case then I should add cases for `x86` and `i386`, yeah? Are there any other architecture types that we support?

Reply:
Sean let me know that the supported 32-bit architectures are listed in the `package.json`. I'll add the cases `x86`, `i386`, as well as `i686`.